docs/pages/router/reference/api-routes.mdx

```diff
@@ -447,6 +447,8 @@ The newer version of the **vercel.json** does not use `routes` and `builds` conf
 
 <Step label="3">
 
+_Note: this step only applies to users of the **legacy** version of the **vercel.json**. If you're using v3, you can skip this step._
```

Comment:
```suggestion
> **Note:** This step only applies to users of the **legacy** version of the **vercel.json**. If you're using v3, you can skip this step.
```

Let's add a callout for this note instead of italicizing it.